scalatest-test/src/test/scala/org/scalatest/DiagrammedAssertionsSpec.scala

```diff
@@ -272,9 +272,9 @@ class DiagrammedAssertionsSpec extends FunSpec with Matchers with DiagrammedAsse
             """
               |
               |assert(0 == a)
-              |       |    |  |
+              |       | |  |
               |       0 |  3
-              |            false
+              |         false
               |""".stripMargin
           )
```

Comment:
@cheeseng Does it mean Scala2 compiler have somewhere improved the positions? Now Dotty agrees with Scala2 on these tests.